cmd/kubeadm/app/constants/constants.go

```diff
@@ -480,7 +480,7 @@ var (
 		23: "3.5.6-0",
 		24: "3.5.6-0",
 		25: "3.5.6-0",
```

Comment:
See https://github.com/kubernetes/kubernetes/pull/114093.
```suggestion
		22: "3.5.7-0",
		23: "3.5.7-0",
		24: "3.5.7-0",
		25: "3.5.7-0",
```

Reply:
Please check the changes

Reply:
Why not update for k8s 1.22 also?

Reply:
Missed on that now it's done

Reply:
Thanks!